R/statDigitPref_refactoring.R

```diff
@@ -0,0 +1,194 @@
+# ref.:
+
+# M. E. Stokes, C. S. Davis and G. G. Koch
+# Categorical Data Analysis Using the SAS System.
+# 3rd Edition.  Cary, NC: SAS Institute, Inc.
+
+# Kari Kuulasmaa, Hans-Werner Hense and Hanna Tolonen
+# Quality Assessment of Data on Blood Pressure in the WHO MONICA Project
+# MONICA Memo, May 1998
```

Comment:
Leave this here, but we'll move it into the docs once we're done sorting out everything else.